tests/func/test_version.py

```diff
@@ -0,0 +1,42 @@
+import re
+
+from dvc.command.version import CmdVersion
+from dvc.cli import parse_args
+
+
+def test_run_in_repo(dvc_repo):
+    cmd = CmdVersion(parse_args(["version"]))
+    ret = cmd.run_cmd()
+    assert ret == 0
+
+
+def test_run_outside_of_repo(repo_dir):
+    cmd = CmdVersion(parse_args(["version"]))
+    ret = cmd.run_cmd()
+    assert ret == 0
+
+
+def test_info_in_repo(dvc_repo, caplog):
+    cmd = CmdVersion(parse_args(["version"]))
+    cmd.run()
```

Comment:
You could just use
```
from dvc.main import main
assert main(["version"]) == 0
```
instead of manually calling CmdVersion, parse_args and run.